www/pages/web-create-project.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+permalink: /web-create-project
+title: Creating projects and uploading files
+---
+
+There are several methods of creating a new project in Mergin:
+
+## Create a blank project
+
+- To create a new project, select **+ Create** in front of **Projects**
+- A new window will appear:
+  - For **Name** type your project name
+  - For **Project owner** select your username or your organisation
+  - By default the project is private. You can choose the option to make it **Public**
+
+After creating a new project, you should be directed to the project page. To add files to your project:
+
+- Click on **Drage & drop here to click and select file(s) to upload**
+- You can then files or folders to upload
+- Once the files and folders are selected, press **Update**
+
+![Upload files](../images/web/web-project-upload.png)
+
+## Create a project from template
+
+Similar to the above process, but in the new project window, under **Template project** you can select a template from the list.
+
+## Cloning an existing project
+
+To clone and existing project:
+
+- Open the project link in your web browser
+- From the top right of the panel, select **Clone**
```

Comment:
It may not be obvious for some - can we add a screenshot with highlight where is the button?